packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -290,7 +290,9 @@ export const messages = {
   error_parsing_sfdx_project_file:
     "Couldn't parse sfdx-project.json file (%s). Parse error: %s",
   sfdx_cli_not_found:
-    'Salesforce CLI is not installed. Install it from [%s](%s)',
+    'Either Salesforce CLI is not installed or only SF v1 is present. Install the latest CLI version from [%s](%s)',
```

Comment:
I changed the error message because SF v1 does not map sf commands to sfdx and thus cannot be used by our extensions.  The isCliInstalled() function in cliConfiguration.ts uses "which sfdx" for the checking, which means having only SF v1 is caught at the same time as having no CLI installed.